kcd-berlin-2022-06-30/README.md

```diff
@@ -0,0 +1,62 @@
+# Flatcar Container Linux basic intro demo for KCD Berlin, 2022
+
+YAML config contains
+1. user "caddy" set-up
+2. inline HTML and separate GIF for a static website
+3. Systemd unit for running caddy to serve the file
+4. Deactivate automated reboots
+
+## Provisioning Demo
+
+Download a Flatcar release (NOT the current / most recent one; use an older version to demo updates) from [here](https://www.flatcar.org/releases/).
+Don't forget to also download the accompanying `flatcar_production_qemu.sh`; we'll need it for the demo.
+
+Move the image to a new name to keep a pristine version; copy it to the original name to have a working copy.
+```shell
+mv flatcar_production_qemu_image.img flatcar_production_qemu_image.img.pristine
```

Comment:
I have the habit to keep the compressed image to avoid making a copy:
```
bzip2 --decompress --keep flatcar_production_qemu_image.img.bz2
```

(same for the image produced by the SDK actually)

Reply:
It's a trade-off CPU vs. disk space; decompressing the image takes quite a bit of time.